via/models/_mixins.py

```diff
@@ -0,0 +1,16 @@
+from datetime import datetime
+
+from sqlalchemy import func
+from sqlalchemy.orm import Mapped, MappedAsDataclass, mapped_column
+
+
+class CreatedUpdatedMixin(MappedAsDataclass):
```

Comment:
These get added as the last columns, I don't think that the case in SQL1.4, not an issue.

Reply:
Yeah. There is actually a [`sort_order` argument to `mapped_column()`](https://docs.sqlalchemy.org/en/20/orm/mapping_api.html#sqlalchemy.orm.mapped_column.params.sort_order) that we can potentially use to control the column order.

Actually, I wonder if we do need to fix this? Say in future we add a new `foo` column to the bottom of this model. After running the DB migration to add `foo` the order of the columns will be `id`, `type`, `video_id`, `title`, `created`, `updated`, `foo`. But if you re-created the DB and SQLAlchemy re-created the table from scratch then the order of the columns would be `id`, `type`, `video_id`, `title`, `foo`, `created`, `updated` :nervous:

Reply:
This got moved into https://github.com/hypothesis/via/pull/1072 (which this PR is now based on) and changed to put the `created` and `updated` columns at the start of the table